jni/Android.mk

```diff
@@ -51,7 +51,6 @@ processors/dcoffsetfilter.cpp \
 processors/decimator.cpp \
 processors/delay.cpp \
 processors/filter.cpp \
-processors/finalizer.cpp \
 processors/flanger.cpp \
```

Comment:
Finalizer was silly, it was basically a limiter with a hard cap for signals outside of the -1 to +1 float range.